Story/Episode 5/st_056150.csv

```diff
@@ -0,0 +1,24 @@
+st_056150_0010#0,"""Aika the Misunderstood, one of the Four Infernal Guardians of [Apprentice].<br>Though shaken by the severe appearance of their friend,<br> the Guardian overcame her, and freed Aika."""
+st_056150_0020#0,"""The group resumed their march. Crushing any Devil Castles in their path,<br>they made their way to the Northern Fortress.<br>However, a heavy doubt slowed their progress."""
+st_056150_0030#0,"""The enemy’s strength is to be feared, surely,<br>but just how fearsome a circumstance would await them next?"""
+st_056150_0040#0,"""The coming conflicts would be nothing like their past battles.<br>They would need to summon up a new kind of courage."""
+st_056150_0050#0,"""It would seem you’ve bested Suzuki somehow..."""
+st_056150_0060#0,"""Hmhmm...However! She was the weakest one among us!"""
+st_056150_0070#0,"""Not to besmirch our great name, but I am many leagues stronger than she!"""
```

Comment:
Try to use a less complex word here, our target english proficiency is grade 9 english.